aws-dynamodb-table/table.tf

```diff
@@ -0,0 +1,38 @@
+resource "aws_dynamodb_table" "this" {
+  name         = var.table_name
+  billing_mode = "PAY_PER_REQUEST"
+  hash_key     = var.hash_key
+  range_key    = var.range_key != "" ? var.range_key : null
+
+  dynamic "attribute" {
+    for_each = var.attributes
+    content {
+      name = attribute.value["name"]
+      type = attribute.value["type"]
+    }
+  }
+
+  dynamic "global_secondary_index" {
+    for_each = var.global_secondary_indexes
+    content {
+      name            = global_secondary_index.value.name
+      hash_key        = global_secondary_index.value.hash_key
+      range_key       = lookup(global_secondary_index.value, "range_key", null)
+      projection_type = global_secondary_index.value.projection_type
+      read_capacity   = lookup(global_secondary_index.value, "read_capacity", null)
+      write_capacity  = lookup(global_secondary_index.value, "write_capacity", null)
```

Comment:
Is this more readable with the `null` fallback or do you think it would look better without it?

Reply:
Hm, já ten lookup moc nepoužíval, ale vypadá to že by tady mohl dávat smysl. 👍